metrics/Makefile

```diff
@@ -11,6 +11,5 @@ release:
 	mkdir -p $(RELEASE_PATH)/alerting-rules
 	$(CP) -r ./examples/kafka/* $(RELEASE_PATH)/
 	$(CP) -r ./examples/grafana/*.json $(RELEASE_PATH)/grafana-dashboards/
-	$(CP) -r ./examples/prometheus/alerting-rules.yaml $(RELEASE_PATH)/alerting-rules/
```

Comment:
How do we now package the YAML files into the release? Also on line 11 we still create the directory.